UNRELEASED.md

```diff
@@ -1,3 +1,5 @@
 # Unreleased
 
-_Nothing yet..._
+**Breaking:**
```

Comment:
This is a fix as opposed to breaking. Yes we are removing an api but it never worked so it cant possibly break anyones code right?

Reply:
Thinking was that we're removing the `onBarTouch` event which some consumers have provided, but as you've said before it would have no effect anyways, so will change to `fixed` :+1: